linkerd/app/admin/Cargo.toml

```diff
@@ -0,0 +1,34 @@
+[package]
+name = "linkerd-app-admin"
+version = "0.1.0"
+authors = ["Linkerd Developers <cncf-linkerd-dev@lists.cncf.io>"]
+license = "Apache-2.0"
+edition = "2018"
+publish = false
+description = """
+The linkerd proxy's admin server.
+"""
+
+[dependencies]
+html-escape = "0.2"
+http = "0.2"
+hyper = { version = "0.14", features = ["http1", "http2"] }
+futures = "0.3"
+linkerd-app-core = { path = "../core" }
+linkerd-app-inbound = { path = "../inbound" }
```

Comment:
out of curiosity, do we want to consider factoring out the shared target types or adding new ones in the admin crate to remove this dependency? definitely makes sense to save that for a future PR; just wondering if it was planned...

Reply:
For now I think this is okay. I _don't_ want to factor out target types -- I generally don't like having "general" target types, they should be pretty narrowly scoped. Treating the admin as another type of inbound is fine today but I think we'll eventually want separate target types (and metrics, etc). But I don't think those changes buy us much in the short term.